hw/hw13/trigrams.py

```diff
@@ -0,0 +1,66 @@
+import random
+
+
+def simplify(lines):
+    # turn a list of lines into a list of words
+    wordlist = []
+    for i in range(len(lines)):
+        wordlist.extend(lines[i].split())
+    return wordlist
+
+
+def make_trigram_dict(words):
+    # turn a list of words into a trigram dictionary
+    d = {}
```

Comment:
Try and avoid single letter variable names. Debugging a bunch of single character variables can be a real pain as your programs become large.